src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -17,4 +17,12 @@ public abstract class Command {
      */
     public abstract CommandResult execute(Model model) throws CommandException;
 
+    /**
+     * Returns true if the command modifies the model in a way that should be saved to storage.
+     *
+     * @return boolean indicating if the command modifies the model.
```

Comment:
Note the `@return` could be omitted as it is the same as the summary statement, but it is also fine to leave it.
```suggestion
     * @return True if the command potentially modifies the model, otherwise false.
```